loadtest/evm.go

```diff
@@ -152,16 +140,17 @@ func (txClient *EvmTxClient) ResetNonce() error {
 	return nil
 }
 
+// nolint
 func withRetry(callFunc func() error) (bool, error) {
```

Comment:
is this used anywhere else than getReceipt?

Reply:
No we can remove it